app/views/admin/calendar/show.html.slim

```diff
@@ -0,0 +1 @@
+= render "admin/calendar/calendar"
```

Comment:
Is this template being used? Not seeing a show action under calendar controller.